### DISCUSSION TITLE
一个Markdown小问题

### DISCUSSION THREAD

**Discussion by leozjr** [obsidianzh/forum#52] [General综合]
如果在obsidian里这样打：

- some words
 -

这个无序列表的元素会被加粗放大（第二个'-'缩进一个tab，删掉后面的空格）
这是BUG还是什么？？

**Comment by makerjackie** [comment]
你用的默认CSS吗？

**Reply by leozjr** [reply]
是的，默认的CSS，刚才发现github怎么传图片😁
![image](https://user-images.githubusercontent.com/44260371/107119438-80df6400-68c2-11eb-9548-dd75e5069406.png)

**Reply by makerjackie** [reply]
hmmm 复现不了你说的问题，我这边挺正常的。（不过加粗语法是这样 `**加粗**`）
